src/components/Counter.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+type State = {
+  count: number,
+};
+
+export class Counter extends React.Component<{}, State> {
+  state: State = {
+    count: 0,
+  };
+
+  addOne = () => {
+    this.setState((state) => {
+      return {
+        count: state.count + 1,
+      };
+    });
+  };
+
+  add100 = () => {
+    this.setState((state) => {
+      return {
+        count: state.count + 100,
+      };
+    });
+  };
+
+  increase = () => {
+    this.addOne();
+
+    if (this.state.count % 5 === 0) {
+      this.add100();
+    }
+
+    return this.state;
+  };
+
+  render() {
+    const { count } = this.state;
+
+    return (
+      <>
+        <h1>
+          Count:
+          {' '}
+          {count}
```

Comment:
`
```suggestion
          {`Count: ${count}`}
```
i think it looks better